modules/common/src/multi-jvm/scala/surge/kafka/KafkaPartitionShardRouterActorMultiJvmSpec.scala

```diff
@@ -91,5 +95,29 @@ class KafkaPartitionShardRouterActorSpecBase
         probe.expectMsg(command0)
       }
     }
+
+    "Forward message and fail" in {
```

Comment:
What's the value in this test? It demonstrates the bug we fixed awhile back but I don't think it catches any regressions in the actual code around that?

Reply:
Initially, I've tried to reproduce it locally, but it works correctly(in my case all actor lifecycle name was the same for multiple nodes). So I've added this test, just to demonstrate the bug as you said

Reply:
Oh gotcha, the original bug was fixed but we just needed something in place to help catch regressions. I think we can get rid of this test here since it's not actually testing for the regression, just demonstrating the original bug